functions/pubsub/main_test.py

```diff
@@ -13,16 +13,18 @@
 # limitations under the License.
 
 import base64
-import flask
-from mock import MagicMock
 import os
 
+from mock import MagicMock
+import pytest
+
 import main
 
 
 FUNCTIONS_TOPIC = os.getenv("FUNCTIONS_TOPIC")
 
 
+@pytest.mark.skip("broken")
```

Comment:
Can we add a link to the issue tracking these samples?

Reply:
#3775